RHIDP-6504: Modularize modules/dynamic-plugins/proc-topology-configure (#1065)
redhat-developer/red-hat-developers-documentation-rhdh: modules/dynamic-plugins/proc-viewing-tekton-pipelineruns.adoc

[id="proc-viewing-tekton-pipelineruns_{context}"]
= Viewing Tekton PipelineRuns

> To view the Tekton PipelineRuns you must grant read access to the `pipelines`, `pipelinesruns`, and `taskruns` resources in the `ClusterRole`:
> 
> [source,yaml]
> ----
>  ...
>   apiVersion: rbac.authorization.k8s.io/v1
>   kind: ClusterRole
>   metadata:
>     name: backstage-read-only
>   rules:
>     ...
>     - apiGroups:
>         - tekton.dev
>       resources:
>         - pipelines
>         - pipelineruns
>         - taskruns
>       verbs:
>         - get
>         - list
> ----
> 
> To view the Tekton PipelineRuns list in the side panel and the latest PipelineRuns status in the Topology node decorator, you must add the following code to the `kubernetes.customResources` property in your `{my-app-config-file}` file:
> 
> [source,yaml]

Reviewer: ```suggestion
.Procedure
. To view the Tekton PipelineRuns, grant read access to the `pipelines`, `pipelinesruns`, and `taskruns` resources in the `ClusterRole`:
+
[source,yaml]
----
 ...
  apiVersion: rbac.authorization.k8s.io/v1
  kind: ClusterRole
  metadata:
    name: backstage-read-only
  rules:
    ...
    - apiGroups:
        - tekton.dev
      resources:
        - pipelines
        - pipelineruns
        - taskruns
      verbs:
        - get
        - list
----

. To view the Tekton PipelineRuns list in the side panel and the latest PipelineRuns status in the Topology node decorator, add the following code to the `kubernetes.customResources` property in your `{my-app-config-file}` file:
+
[source,yaml]
```
